carbon-design-system/carbon
# fix: 10502-radio-button-misalignment-bug

Closes #10502 

The error message in the radio button overlaps the label.

#### Changelog

**New**

- Changed some styles

#### Testing / Reviewing

- Check If the error message does not overlap the radio button label by pressing the submit and make sure none of the radio button is selected to trigger the error message.
- Check test storybook under RadioButton / Test: https://deploy-preview-15435--v11-carbon-react.netlify.app/?path=/story/components-radiobutton--test
- Please make sure to remove Test storybook before merging
